build/build-allocation-images/go/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG BASE_IMAGE=agones-build-allocator-base:latest
```

Comment:
Where do you define this base image (`agones-build-allocator-base:latest`)? In `build/build-sdk-images/go/Dockerfile` it extends `agones-build-sdk-base` but that image is at https://github.com/googleforgames/agones/blob/840e5405b3c05b144f33fa97798461d53ccfd558/build/includes/sdk.mk#L92. I don't see anything in the makefile that builds the base image, just this file that extends it.